modules/nixos/main.py

```diff
@@ -672,13 +397,13 @@ def run():
         gs.value("packagechooser_packagechooser") is not None
         and gs.value("packagechooser_packagechooser") != ""
     ):
-        cfg += cfgmisc
+        cfg += getCustomOrDefaultSnippet('misc')
```

Comment:
in contrast to the original `cfgmisc` the audio settings are missing in `misc` snippet.
```suggestion
        cfg += getCustomOrDefaultSnippet('misc')
        cfg += getCustonOrDefaultSnippet('audio')
```